StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1202UnitTests.cs

```diff
@@ -394,6 +526,21 @@ public void B()
             var expected = this.CSharpDiagnostic().WithLocation(7, 17).WithArguments("public", "methods", "private");
 
             await this.VerifyCSharpDiagnosticAsync(testCode, expected, CancellationToken.None).ConfigureAwait(false);
+
+            var fixedCode = @"public class TestClass
+{
+
+    public void B()
+    {
+    }
+    private static void A()
+    {
+    }
+}
```

Comment:
The code fix should preserve the empty line between the methods, as this will trigger SA1516 after executing the code fix.